WolvenKit.App/ViewModels/Documents/RDTDataViewModel.cs

```diff
@@ -473,21 +474,23 @@ public void SetSelection(List<ChunkViewModel> chunks)
         {
             return;
         }
-
+        
         // remove duplicates
         var uniqueChunks = new HashSet<ChunkViewModel>(chunks);
-        SelectedChunk = uniqueChunks.LastOrDefault();
-
         foreach (var cvm in uniqueChunks)
         {
-            cvm.IsSelected = true;
+            // cvm.IsSelected = true;
```

Comment:
why comment this out?

Reply:
because it works this way. Should have deleted it

Reply:
yeah but where is the IsSelected property then set?

Reply:
It happens implicitly via binding, I think? All this stuff is too nested >.< I guess commenting this back in won't hurt, I'll test after work :)

Reply:
nah I just wanted to know the reason for this change... Yeah I guess it is set through the OnSelected Event. lemme check

Reply:
And? :D